launcher/src/main/java/org/apache/spark/launcher/JavaModuleOptions.java

```diff
@@ -37,7 +37,8 @@
       "--add-opens=java.base/sun.nio.ch=ALL-UNNAMED",
       "--add-opens=java.base/sun.nio.cs=ALL-UNNAMED",
       "--add-opens=java.base/sun.security.action=ALL-UNNAMED",
-      "--add-opens=java.base/sun.util.calendar=ALL-UNNAMED"};
+      "--add-opens=java.base/sun.util.calendar=ALL-UNNAMED",
+      "--add-opens=java.security.jgss/sun.security.krb5=ALL-UNNAMED"};
```

Comment:
Not sure how important is this but on IBM JVM this is called `com.ibm.security.krb5`.
Since there are several `sun...` options up I presume it doesn't matter.